client/signup/steps/woocommerce-install/hooks/use-site-settings/index.tsx

```diff
@@ -81,22 +81,25 @@ export function useSiteSettings( siteId: number ) {
 	 * The data will be saved to the remote server.
 	 */
 	const save = useCallback( () => {
-		if ( ! editedSettings || ! editedSettings.length ) {
-			return;
-		}
-
-		/*
-		 * Save the edited options to the server.
-		 * After the save is complete, clean the private store.
-		 */
-		const newSettings = editedSettings.reduce(
-			( acc, settingKey ) => ( { ...acc, [ settingKey ]: settings[ settingKey ] } ),
-			{}
-		);
-
-		dispatch( saveSiteSettings( siteId, newSettings ) );
-		setEditedSettings( [] );
-	}, [ dispatch, editedSettings, settings, siteId ] );
-
-	return { save, update, get };
+		setEditedSettings( ( state ) => {
```

Comment:
This doesn't work just wanted to get something up to explain the problem better, I thought this was working for a bit but it was due to a completed:true value leaking into existing state from a previous fix attempt that only worked in some situations.